src/resources.rs

```diff
@@ -186,6 +187,8 @@ pub use {
 #[rustfmt::skip]
 #[cfg(feature = "checkout")]
 pub use {
+    checkout::checkout_session_ext::*,
```

Comment:
I saw a lot of `use` blocks with `*_ext::*` for respective entities but `checkout` didn't have one – now it does. Should we keep it?

Reply:
Since `promotion_code_ext::*` was moved to `#[cfg(feature = "products")]`, there was no reason to include `checkout_session_ext::*` in this PR. Removed.